test-utils/src/main/scala/com/mesosphere/utils/mesos/FaultDomain.scala

```diff
@@ -0,0 +1,11 @@
+package com.mesosphere.utils.mesos
```

Comment:
Shouldn't this be part of some models? I don't mean `core-models` but `com.mesosphere.utils.models`.

Reply:
I'm not starting this discussion _again_. Once we get to the point when they are actually needed somewhere, we can revisit this decision.